service/commentutil/commentutil.go

```diff
@@ -0,0 +1,52 @@
+package commentutil
+
+import (
+	"fmt"
+
+	"github.com/reviewdog/reviewdog"
+)
+
+// `path` to `position`(Lnum for new file) to comment `body`s
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
comment on exported type PostedComments should be of the form "PostedComments ..." (with optional leading article)